volontulo/templates/volontulo/show_offer.html

```diff
@@ -3,16 +3,21 @@
 {% block title %}Lista ofert Volontulo{% endblock %}
 
 {% block sidebar %}
-<ul class="list-unstyled">
-    <li><a href="{% url 'show_offer' 1 %}">Oferta 1</a></li>
-    <li><a href="{% url 'show_offer' 2 %}">Oferta 2</a></li>
-    <li><a href="{% url 'show_offer' 3 %}">Oferta 3</a></li>
-    <li><a href="{% url 'show_offer' 4 %}"><strong>Oferta 4</strong></a></li>
-    <li><a href="{% url 'show_offer' 5 %}">Oferta 5</a></li>
-    <li><a href="{% url 'show_offer' 6 %}">Oferta 6</a></li>
-    <li><a href="{% url 'show_offer' 7 %}">Oferta 7</a></li>
-    <li><a href="{% url 'show_offer' 8 %}">Oferta 8</a></li>
-</ul>
+    <div class="row-xs-3">
```

Comment:
why this `div`? it would have to be directly in a `.row` element and the columns arealready defined in the col2.html template
Or am I missing something?